modules/rosa-planning-considerations.adoc

```diff
@@ -1,22 +1,25 @@
 
 // Module included in the following assemblies:
 //
-// rosa_planning/rosa-planning-environment.adoc
+// rosa_planning/rosa-limits-scalability.adoc
 
-[id="initial-planning-considerations_{context}"]
-= Initial planning considerations
+[id="control-plane-and-infra-node-sizing-and-scaling_{context}"]
+= Control plane and infrastructure node sizing and scaling
 
-Consider the following tested object maximums when you plan your {product-title} cluster.
+When you install a {product-title} (ROSA) cluster, the number of required control plane and infrastructure nodes is automatically determined by the compute node count.
 
-These guidelines are based on a cluster of 102 workers in a multi-availability zone configuration. For smaller clusters, the maximums are lower.
+If you change the number of compute nodes in your cluster after installation, the Red Hat Site Reliability Engineering (SRE) team scales the control plane and infrastructure nodes as required, to maintain cluster stability.
```

Comment:
Optional: 
s/as required, to maintain/as required to maintain

Reply:
Thank you for this suggestion. Resolved.